index.html

```diff
@@ -1,20 +1,22 @@
 <!doctype html>
   <html>
-    
-   <!--bootstrap-->
-    <link rel="stylesheet" link="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
+  <head>
+      <!--bootstrap-->
+       <link rel="stylesheet" link="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
This feels a bit... off to me. Want to compare to the documentation for the link tag and see if you can resolve this? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link